exercises/exercise3.js

```diff
@@ -1,12 +1,126 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Say Hello to Babushka!',
+    prompt: "Hello, Babushka",
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Privyet! What Babushka can do for you?';
     },
+    isReply: true,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+
+  {
+    label: 'censorship',
+    prompt: "#{name1}",
```

Comment:
The way that the hubot code was written on the back end it does not like or does not know how to compute a prompt that only has a param without anything in front of it. So, "#{name}" causes an error in the bot load, I added a word before each of those to get it working.